pallets/multicollateral-bonding-curve-pool/Cargo.toml

```diff
@@ -27,6 +27,7 @@ serde = { version = "1.0.101", default-features = false, optional = true, featur
 sp-arithmetic = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
 sp-runtime = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
 sp-std = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
+sp-core = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
```

Comment:
```suggestion
```
No need in this dep, it is used to construct AssetId in bench. But there is more convenient way to do it.